index.html

```diff
@@ -147,7 +151,27 @@
       /**
       * Makes the divs for initial setup
       */
+
+      // thanks for code on converting txt file to js array: https://www.codingforums.com/javascript-programming/270314-javascript-code-read-text-file-array.html#post_1260248
+
+    /*probably needs updating though because of firefox console error: 
+
+    Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user’s experience. For more help http://xhr.spec.whatwg.org/
```

Comment:
I'm getting the same error with Chrome when trying to run your updated code locally. So far I'm only getting a blank page - did you do anything else to make it display?
Is there something about needing to run a separate "server" for the data to avoid this error?

Reply:
Yeah I'll look at it more soon -- I haven't tried it on chrome. May be that we just won't want to use a data.txt with js, seems weirdly tricky (I need to learn more about how js works in general, still pretty mysterious to me)